prow/cmd/deck/static/index.html

```diff
@@ -13,7 +13,7 @@
     <body>
         <header>
             <ul>
-                <li><a href="/"><h1>Prow Status</h1></a></li>
+                <li><a href="https://github.com/kubernetes/test-infra" class="logo"><img src="/logo.svg" alt="kubernetes logo" class="logo"></img></a><a href="/" class="current"><h1>Prow Status</h1></a></li>
```

Comment:
I think it might make more sense for the kubernetes logo to link to `kubernetes/test-infra/prow` or `kubernetes/kubernetes`.

Reply:
Linking to Prow SGTM